tests/test_linalg/test_solvers/test_belief_updates/test_matrix_based/test_matrix_based_linear_belief_update.py

```diff
@@ -93,7 +93,8 @@ def dense_matrix_based_update(
     np.testing.assert_allclose(
         updated_belief.A.cov.todense(),
         A_naive.cov.todense(),
-        err_msg="Covariance of system matrix estimate does not match naive implementation.",
+        err_msg="""Covariance of system matrix estimate does not
+        match naive implementation.""",
```

Comment:
```suggestion
        err_msg=(
            "Covariance of system matrix estimate does not match naive implementation."
        ),
```
since we don't really want a line break in the error message